projects/Mallard/src/helpers/push-download-failsafe.ts

```diff
@@ -0,0 +1,39 @@
+import BackgroundFetch from 'react-native-background-fetch'
+import { pushTracking } from './push-tracking'
+import { clearAndDownloadIssue } from './download-todays-issue'
+
+const pushDownloadFailsafe = () => {
```

Comment:
should it be called something like `setupBackgroundDownload` rather than a noun? (failsafe)

Reply:
I wanted to make it very specific. This is the failsafe for the push download. Happy to change if it's not clear.